loot_importer.user.js

```diff
@@ -0,0 +1,574 @@
+      'color': 'red',
+      'margin-top': '4px',
+      'margin-bottom': '4px'
+    });
+    mbContentBlock.prepend(warning_buggy);
+  }
+
+  // Form parameters
+  var edit_note = MBImport.makeEditNote(window.location.href, 'Loot');
+  LOGGER.debug("*** Edit Note: ", edit_note);
+  var parameters = MBImport.buildFormParameters(release, edit_note);
+  LOGGER.debug("***Form parameters");
+  // Build form + search button
+  var innerHTML = '<div id="mb_buttons">' + MBImport.buildFormHTML(parameters) + MBImport.buildSearchButton(release) + '</div>';
+  mbContentBlock.append(innerHTML);
+
+  insertMbUI(mbUI);
+
+  $('#mb_buttons').css({
+    display: 'inline-block',
+    width: '100%'
+  });
+  $('form.musicbrainz_import').css({
+    width: '49%',
+    display: 'inline-block'
+  });
+  $('form.musicbrainz_import_search').css({
+    'float': 'right'
+  })
+  $('form.musicbrainz_import > button').css({
+    width: '100%',
+    'box-sizing': 'border-box'
+  });
+
+  mbUI.slideDown();
+}
+
+
+// Analyze Loot data and return a release object
+function ParseLootPage() {
+  LOGGER.debug("ParseLootPage function firing");
+
+  var releasebarcode = "";
+  var releasecountry = "";
+  var releasedaterel = "";
+  var releaselanguage = "";
+  var releasetitle = "";
+  var releaseartist = "";
+  var prodlabels = [];
+  var release_artist_array = [];
+  var release_format = "";
+
+  // div#productContent table tbody tr.productOverview td.productInfo h1
+  var AlbumName = document.querySelectorAll("div#productContent > table > tbody > tr.productOverview > td.productInfo > *");
+  LOGGER.debug("Album Name: ", AlbumName[0].innerText); // textContent give the space formatting and we just need the text so we use innerText
+  releasetitle = AlbumName[0].innerText;
+  LOGGER.debug("Artist Name: ", AlbumName[1].innerText);
+  releaseartist = AlbumName[1].innerText;
+
+  // select the product info
+  var ReleaseInfo = document.querySelectorAll("div#tab-1 > table > tbody > tr > *");
+  LOGGER.debug("Product Info: ", ReleaseInfo);
+
+  for (var prodinfoloop = 0; prodinfoloop < ReleaseInfo.length; prodinfoloop++) {
+    var prodinfolabel = ReleaseInfo[prodinfoloop];
+    //LOGGER.debug("** ProdInfo LABEL: ",prodinfolabel);
+    if (prodinfolabel.tagName == "TD") {
+      var prodinfolabellowcase = prodinfolabel.innerText.toLowerCase().trim(); //replaced textContent with innerText
+
+      LOGGER.debug("<TD>", prodinfolabellowcase);
+
+      switch (prodinfolabellowcase) {
+        case "label:": // use these cases to select the spesific text values
+          LOGGER.debug("* label: selected *");
+          var release_label = new Object();
+          release_label.name = prodinfolabel.nextElementSibling.innerText.trim();
+          prodlabels.push(release_label);
+          break;
+        case "release date:": // "release date:" TODO: NOT WORKING
```

Comment:
Please assist as case "release date:" is never matched and I can't see the fault